### ISSUE TITLE
Brother, can you add that I can send messages to users and whatever I want? 😂😂 I will message my friends and play a prank on them

### ISSUE THREAD

**Issue by h-ug** [Emperor-One/Hamster-Key-Telegram-Bot#20]

**Comment by Emperor-One** [comment]
Telegram doesn't allow you to do that. Users, first, need to "start" the bot and consent to being sent messages while using the bot, only then will you be able to send them messages.

**Comment by h-ug** [comment]
> Telegram doesn't allow you to do that. Users, first, need to "start" the bot and consent to being sent messages while using the bot, only then will you be able to send them messages.

Yes I know that do it for us please make the bot save all users in a database then when you want to send a message to them you can do it
